apps/client/src/app/[user]/components/LeftSidebar/SidebarProvider/hook.ts

```diff
@@ -0,0 +1,10 @@
+import { useContext } from "react";
+import { SidebarContext } from ".";
+
+export const useSidebrChip = () => {
```

Comment:
```suggestion
export const useSidebarChip = () => {
```

spell check extension 설치해두면 좋을 것 같아요!

Reply:
(대충 개발환경 탓하는 내용)
제 pc엔 설치돼 있어서 괜찮았었는데.. 여기서도 설치해서 사용할게요. 꼼꼼히 봐줘서 고마워요.